{
    "observer": {
        "repo": "https://github.com/ansible/Ansible",
        "labels": ["affects_2.7", "azure", "bug", "cloud", "new_contributor", "python3"],
        "credential_name": "github_token"
    },
    "dataStore": {
        "repo": "https://github.com/VSChina/ansible-issue-tracker",
        "branch": "state",
        "filePath": "issue-state.json",
        "credential_name": "github_token"
    },
    "monitor": {
        "repo": "https://github.com/VSChina/ansible-issue-tracker",
        "credential_name": "github_token"
    }, 
    "custom": {
        "maintainers":  ["kyliel", "yuwzho", "zikalino", "Fred-sun", "yungezz"],
        "mails": ["yuwzho@microsoft.com", "37337818+yungezz@users.noreply.github.com", "zikalino@microsoft.com"],
        "ignoreLabel": ["wontfix"]
    }
}